src/processor/handler.rs

```diff
@@ -181,12 +181,12 @@ pub trait Context {
 }
 
 #[derive(Clone)]
-pub struct TransactionContext {
+pub struct ZmqContext {
```

Comment:
I'm not sure if this should be public anymore, since SDK users should not be creating one of these directly

Reply:
Done